tests/fixtures/weather_html_hour.yml

```diff
@@ -0,0 +1,854 @@
+http_interactions:
```

Comment:
In this file the http request and response is saved. A cassette could contain several several requests and responses if the code inside "use_cassette()" made several requests.

Reply:
Note that the diff of cassettes can be hidden if one edits the `.gitattributes` of the repo.

```
* text=auto

tests/fixtures/**/* -diff
```